src/content/edit-user-script.js

```diff
@@ -147,7 +147,17 @@ function onSave() {
 editor.on('swapDoc', doc => {
   if (doc.getMode().name == 'javascript') {
     doc.setOption('gutters', ['CodeMirror-lint-markers']);
-    doc.setOption('lint', true);
+    doc.setOption('lint', {
+      formatAnnotation(ann) {
+        if (ann.from.ch == 4) {
```

Comment:
Why 4?

Is there a named constant available that would make this code more readable?

Reply:
I'm not 100%, but I think the 4 is the column number of the `@` sign used for keywords.

Reply:
It's the col number. I moved this to the lint function since it's meant for metadata.